### PR TITLE
[conflict_resolver] scoring algorithm error message

### PR DESCRIPTION
**Pull request by shonibare**
Added instrument name to the error message for clarity when it failed on conflict resolver #7701

### CODE REVIEW

File: modules/instrument_builder/jsx/react.questions.js [hunk 1]
```diff
@@ -916,7 +916,7 @@ class AddElement extends Component {
       if (minDate > maxDate && min !== '' && max !== '') {
         let temp = (this.state.error) ? this.state.error : {};
 
-        temp.dateOption = 'End year append before start year';
+        temp.dateOption = 'End year happened before start year';
```
Comment (driusan) [inline modules/instrument_builder/jsx/react.questions.js:919]: I think this needs to be rebased, this was a different PR.

Comment (shonibare) [reply modules/instrument_builder/jsx/react.questions.js:919]: It was a previous PR that had already been merged. Have been trying to rebase or close it and issue a new PR but when I do so my PR's keep committing to the main branch while I made changes on my forked, I even used cherry-pick @driusan @ridz1208 what do you suggest I should do?

Comment (shonibare) [reply modules/instrument_builder/jsx/react.questions.js]: I have done the rebase @ridz1208

**Comment by ridz1208** [comment]
@shonibare thanks

**Comment by ridz1208** [comment]
@driusan The Pr was not rebased but main was merged into it. the squashing will take care of the rest. its all urs

**Review by ridz1208 — APPROVED** [review]

**Pull request merged**